src/cftlib/resources/scripts/start.sh

```diff
@@ -1,19 +1,3 @@
 #!/bin/bash
 
```

Comment:
maybe we just need a small comment here to indicate to the dev that this scripting has been deprecated and that they should view the README